src/pushsource/_impl/model/container.py

```diff
@@ -317,3 +317,11 @@ class OperatorManifestPushItem(PushItem):
 
     :type: List[str]
     """
+
+    labels = attr.ib(type=dict, default=attr.Factory(frozendict), converter=frozendict)
+    """Labels of this image, if known.
```

Comment:
What image?  This item represents an operator_manifests.zip, is it correct to call that an "image"?

Reply:
Way I see it OperatorManifestPushItem  represents whole container, not just one archive. Also technically labels are not associated with the operator_manifests.zip archive, but with other archives belonging to the container.